### PR TITLE
AfterEffects: make new publisher default

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Removed old Pyblish and switched to `Publisher`

## Description
Updated documentation, updated extension menu

## Additional info
Needs to redeploy extension!!!
Refactored to new host class to standardize.

## Testing notes:
1. try various publishing in AE
2. try to publish workfile with instances created in Pyblish

### CODE REVIEW

**Review by 64qam — APPROVED** [review]

File: openpype/hosts/aftereffects/plugins/create/create_render.py [hunk 1]
```diff
@@ -23,12 +24,13 @@ def __init__(self, project_settings, *args, **kwargs):
                                                   ["create"]
                                                   ["RenderCreator"]
                                                   ["defaults"])
+        self._host = AfterEffectsHost()
```
Comment (iLLiCiTiT) [inline openpype/hosts/aftereffects/plugins/create/create_render.py:27]: Don't create new instance of object. BTW creators have access to `self.host`, you don't have to create it at all.

File: openpype/hosts/aftereffects/api/pipeline.py [hunk 2]
```diff
@@ -30,27 +37,155 @@
...
+
+        stub.remove_instance(inst_id)
+
+        if instance.get("members"):
+            item = stub.get_item(instance["members"][0])
+            if item:
+                stub.rename_item(item.id,
+                                 item.name.replace(stub.PUBLISH_ICON, ''))
+
+    def uninstall(self):
```
Comment (iLLiCiTiT) [inline openpype/hosts/aftereffects/api/pipeline.py:185]: I would remove `uninstall` it was never used.

File: openpype/hosts/aftereffects/plugins/create/create_render.py [hunk 3]
```diff
@@ -46,7 +47,7 @@ def update_instances(self, update_list):
 
     def remove_instances(self, instances):
         for instance in instances:
-            api.remove_instance(instance)
+            self._host.remove_instance(instance)
```
Comment (iLLiCiTiT) [inline openpype/hosts/aftereffects/plugins/create/create_render.py:50]:
```suggestion
            self.host.remove_instance(instance)
```

File: openpype/hosts/aftereffects/api/pipeline.py [hunk 4]
```diff
@@ -30,27 +36,150 @@
...
+                return item
+
+        return {}
+
+    def update_context_data(self, data, changes):
+        item = data
+        item["id"] = "publish_context"
+        self.stub.imprint(item["id"], item)
+
+    def get_context_title(self):
```
Comment (iLLiCiTiT) [inline openpype/hosts/aftereffects/api/pipeline.py:123]: `HostBase` already implement this functionality for you.

**Review by iLLiCiTiT — APPROVED** [review]
Looks good

**Review by 64qam — APPROVED** [review]

**Pull request merged**